js/jquery-comments.js

```diff
@@ -594,13 +594,23 @@
         },
 
         updateToggleAllButton: function(parentEl) {
+            // Don't hide replies if maxRepliesVisible has negative value
+            if (this.options.maxRepliesVisible < 0) return;
```

Comment:
I thinks it's more intuitive to use null or undefined for showing all the replies:

 if (!this.options.maxRepliesVisible) return;